src/subscriptions/restHookLambda/restHook.ts

```diff
@@ -2,6 +2,8 @@ import axios from 'axios';
 import { makeLogger } from 'fhir-works-on-aws-interface';
 import { SQSEvent } from 'aws-lambda';
 import { SubscriptionNotification } from 'fhir-works-on-aws-search-es';
+import { metricScope, Unit } from 'aws-embedded-metrics';
+import moment from 'moment';
```

Comment:
I'd suggest not using `moment`

The difference in milliseconds is easy to calculate using the available Date functions
```
Date.now() - (new Date('2022-02-17T20:33:57.021Z')).getTime()
```

Also, moment is sort of deprecated nowadays: https://github.com/moment/moment/#project-status